BMM.UI.Android/Resources/layout/fragment_share_trackcollection.axml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout 
```

Comment:
fragment_tracklist has pretty much the right design. Maybe it makes sense to reuse it?

Reply:
I think you mean Shared Playlist view where we have list of songs/podcasts and for sure i want to reuse it, but is't not implemented yet. But in this case the fragment is showing Share Playlist view and I don't see any similiar view in the app, that's why I want to create separrate view for it.